.changeset/light-sloths-crash.md

```diff
@@ -0,0 +1,7 @@
+---
+"create-eth": patch
+---
+
+- fix: BigInt parsing losing precision in IntegerInput (https://github.com/scaffold-eth/scaffold-eth-2/pull/893)
+- feat: bundler module resolution (https://github.com/scaffold-eth/scaffold-eth-2/pull/885)
+- fix: ignore strings starting with 0 (https://github.com/scaffold-eth/scaffold-eth-2/pull/894)
```

Comment:
I like a new way of writing changesets!

Reply:
lol I kept it like this assuming it will look like this (the link will shortend in release.md )
![Screenshot 2024-07-17 at 10 09 22 PM](https://github.com/user-attachments/assets/14624d72-a8aa-4530-8905-8f038da8008b)

Reply:
<3 https://github.com/scaffold-eth/create-eth/releases/tag/v0.0.51

![Screenshot 2024-07-17 at 10 19 56 PM](https://github.com/user-attachments/assets/7742a490-dbc7-49c1-948f-5ecd70920ebd)

Reply:
Looks great! 😍
And we need to choose one way to add links :)